src/index_de.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="de">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>ninjafirewall-config-browser</title>
+</head>
+
+<body>
+    <main class="py-4 px-4">
+        <h2 class="text-xl font-bold text-center text-center">
+            Konfigurationsdatei für NinjaFirewall WP Edition
+        </h2>
+        <div class="text-center pb-8">
+            <br/>
+            <p class="max-w-md mx-auto">
+                Bitte denke unbedingt daran, den <strong>Full WAF Mode</strong> sowie ggf. das <strong>Monitoring (File Check and File Guard)</strong> zu aktivieren.
```

Comment:
Was this intended to be partly translated? Just to clarify.

Reply:
As long as NF only available in english, yes. Otherwise Monitoring = Monitoring and (File Check and File Guard) = (Dateisystem Check und Zugriffsüberwachung).

<strong>Monitoring (Dateisystem Check und Zugriffsüberwachung)</strong>

-> Anglismen here and there can be used. (Monitoring, Check)

Reply:
I would argue that File Guard and File Check are trade names of the code parts.
My main point was "and" instead of "und".

```suggestion
                Bitte denke unbedingt daran, den <strong>Full WAF Mode</strong> sowie ggf. das <strong>Monitoring (File Check und File Guard)</strong> zu aktivieren.
```

Reply:
Perfetto.